pages.es/common/doggo.md

```diff
@@ -0,0 +1,25 @@
+# doggo
+
+> Cliente DNS para Humanos.
+> Escrito en Golang.
+> Más información: <https://doggo.mrkaran.dev>.
+
+- Realiza una simple búsqueda DNS:
+
+`doggo {{ejemplo.com}}`
+
+- Consulta registros MX usando un servidor de nombres específico:
+
+`doggo MX {{codeberg.org}} @{{1.1.1.2}}`
+
+- Utiliza DNS sobre HTTPS:
+
+`doggo {{ejemplo.com}} @{{https://dns.quad9.net/dns-query}}`
+
+- Salida en formato JSON:
+
+`doggo {{ejemplo.com}} --json | jq '{{.responses[0].answers[].address}}'`
```

Comment:
```suggestion
`doggo {{example.com}}`

- Consulta registros MX usando un servidor de nombres específico:

`doggo MX {{codeberg.org}} @{{1.1.1.2}}`

- Utiliza DNS sobre HTTPS:

`doggo {{example.com}} @{{https://dns.quad9.net/dns-query}}`

- Salida en formato JSON:

`doggo {{example.com}} --json | jq '{{.responses[0].answers[].address}}'`
```
I think it would be better to add link to example.com instead of the current one because it redirects to a weird domain.